app/helpers/navbar_helper.rb

```diff
@@ -0,0 +1,20 @@
+module NavbarHelper
+  def self.show_navbar?(controller, action, settings)
```

Comment:
The param name `settings` here is ambiguous. From looking at how it's called, i see it comes from `context.system_settings.display_navbar?`. Perhaps we name it `display_navbar` to match? Open to other suggestions as well.

Reply:
Done :+1: